v19.2/learn-cockroachdb-sql-movr.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Learn CockroachDB SQL
```

Comment:
I don't understand why this is a new file named `learn-cockroachdb-sql-movr.md` and not an update to the existing `learn-cockroachdb-sql.md`?  Aha, it looks like you pinged me in #5040 but I didn't get an email, weird.  My opinion is that we should do the minimal update to edit the existing "Learn CockroachDB SQL" page to make it "movr-compatible", since I think making "Learn CockroachDB SQL" a family of pages with different options for starting the database is adding complexity to the "user journey", and using "cockroach demo" vs. spinning up a local cluster is sort of a separate issue we should decide how to address / standardize in our docs IMO.  But let's discuss!  We should def get Jesse's opinion when he returns from vacation, I'm curious about his thoughts also.

Reply:
I think you are right on this... We should be aiming to simplify docs, especially "Learn CockroachDB" docs and others of that ilk. I'll simplify this just as you suggest.